src/Php54/Php54.php

```diff
@@ -39,4 +39,67 @@ public static function hex2bin($data)
 
         return $data;
     }
+
+    public static function http_response_code($responseCode)
+    {
+        static $currentCode = 200;
+
+        $messages = array(
```

Comment:
This array us not necessary: php source already embed the list for you. All you have to do is send a `header('Status: XXX')` and let it do the job